### PR TITLE
Add alerts when add/edit workflows

### PR DESCRIPTION
**Pull request by szyn**
I'm adding alert component when adding / editing workflow using digdag-ui.

* When the workflows failed to store.  

* When the workflows to save.

### CODE REVIEW

File: digdag-ui/console.jsx [hunk 1]
```diff
@@ -1229,6 +1229,17 @@ class FullTimestamp extends React.Component {
   }
 }
 
+const Alerts = ({alertType, message}:{alertType:string, message:string}) => {
+  switch (alertType) {
+    case 'success':
+      return <div className="alert alert-success" role="alert">{message}</div>
```
Comment (dzannotti) [inline digdag-ui/console.jsx]: props should use single quote `'`

**Comment by dzannotti** [comment]
@szyn Thanks for this PR. I left a note, that applies to most PR, but i'm happy to merge this and make the changes myself in a separate PR if you wish, let me know

**Comment by szyn** [comment]
@dzannotti 
Thank you very much for the review. 
I see. Never mind! I will correct that. 
Thank you for your kindness and cooperation.

**Comment by dzannotti** [comment]
@szyn thank you for the pr! If it's of any help `npm run lint` should help you highlight this kind of issues

**Comment by szyn** [comment]
@dzannotti 
Oh, I missed that. Sorry about that. Thank you for telling me the how to use lint. 
I will do that from next time!

File: digdag-ui/console.jsx (hunk 1 above)
Comment (szyn) [inline digdag-ui/console.jsx]: Thank you for your review. 
I corrected the points that were pointed out.

**Pull request merged**